modules/core/src/main/scala/net/BitVectorSocket.scala

```diff
@@ -81,7 +81,7 @@ object BitVectorSocket {
     sslOptions:   Option[SSLNegotiation.Options[F]],
   ): Resource[F, BitVectorSocket[F]] =
     for {
-      sock  <- sg.client[F](new InetSocketAddress(host, port))
+      sock  <- sg.client[F](new InetSocketAddress(host, port), noDelay = true)
```

Comment:
Why not just submitting a PR to Rob Norris?

Reply:
I can but not sure he will republish a 0.0.x (CE2 branch)